### ISSUE TITLE
tool used in preparing training data ?

### ISSUE THREAD

**Issue by ziodos** [wenwenyu/PICK-pytorch#86]
can you provide some details about the method you used to prepare training data , I think you didn't use a classic ocr tool, thanks.

**Comment by knitemblazor** [comment]
you have to use an ocr tool. just mapping the text with corresponding label is an issue i would suggest you use labelImg to get the region and then use overlapping text region to make corresponding labels.

**Comment by ziodos** [comment]
I am using tesseract as text detection and text recognition tool , the author said that it wasn't good for result accuracy , I still don't know why

**Comment by NeerajAI** [comment]
There are issues in tesseract , it does not work with complex document structure and ocr also fails some time.

**Comment by NeerajAI** [comment]
Any idea, how they converted x,y,w,h value into 8 coordinates in traindata annotation files ?
